examples/util/OpenCVRunner.js

```diff
@@ -0,0 +1,165 @@
+class OpenCVRunner extends BaseRunner {
+  constructor() {
+    super();
+    this._output = null;
+  }
+
+  /**
+   * This method is to load model file with specified url.
+   * @param url: A string for model file url.
+   * @returns {string} This returns status of load model file, ['ERROR' | 'SUCCESS'].
+   */
+  _loadModelFile = async (url) => {
+    let status = 'ERROR';
+
+    if (url !== undefined) {
+      const arrayBuffer = await this._loadURL(url, this._progressHandler, true);
+      const bytes = new Uint8Array(arrayBuffer);
+      switch (url.split('.').pop()) {
+        case 'onnx':
+          const err = onnx.ModelProto.verify(bytes);
+          if (err) {
+            throw new Error(`The model file ${url} is invalid, ${err}`);
+          }
+          try {
+            cv.FS_createDataFile('/', this._currentModelInfo.modelId, bytes, true, false, false);
+          } catch (e) {
+            if (e.errno === 17) {
+              console.log(`${this._currentModelInfo.modelId} already exited.`);
+              status = 'SUCCESS'
+            } else {
+              console.error(e);
+            }
+          }
+          break;
+        default:
+          throw new Error(`Unrecognized model format, support TFLite | ONNX | OpenVINO model`);
+      }
+    } else {
+      throw new Error(`There's none model file info, please check config info of modelZoo.`);
+    }
+
+    this._setLoadedFlag(true);
+    return status;
+  };
+
+  /**
+   * This method is to load model with given model info.
+   * @param modelInfo: A string for model info.
+   * @returns {string} This returns 'LOADED' status if model already loaded.
+   */
+  loadModel = async (modelInfo) => {
```

Comment:
Please move the document of `loadModel` to `BaseRunner`. We don't need to replicate that in each sub class.